spec/standard-webhooks.md

```diff
@@ -138,9 +138,9 @@ The unique identifier is a unique identifier associated with a specific event tr
 
 #### Signature scheme
 
-As mentioned above, it's important to sign both the body of the webhook, and the associated metadata. To achieve this, the metadata and the body are concatenated (delimited by full-stops) and then signed.
+As mentioned above, it's important to sign both the body of the webhook and the associated metadata. To achieve this, the message's ID, timestamp and body are concatenated - delimited by full stops - and then signed.
 
-The content to be signed is therefore: `{msg_id}.{timestamp}.{payload}`.
+The content to be signed is therefore: `msg_id.timestamp.payload`.
```

Comment:
I feel like the previous version with the `{}` is more clear. That's a common text replacement convention, no?

Reply:
It might be a common interpolation syntax but it could lead to misinterpretation.